tasks.py

```diff
@@ -71,7 +71,7 @@ def build(ctx, binaries, architectures, tag="dev", docker_user="metallb"):
     binaries = _check_binaries(binaries)
     architectures = _check_architectures(architectures)
     _make_build_dirs()
-    
+
```

Comment:
My editor is set up to strip trailing whitespace and I got tired of removing these changes, so I left them.  Normally I would leave unnecessary formatting changes out completely, but since it was only whitespace-only lines changed, I figured it wasn't too distracting this time.

Reply:
No problem :)